web/tests/test_github_links.py

```
@@ -0,0 +1,70 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from web.models import ForumCategory, ForumTopic
+
+User = get_user_model()
+
+
+class ForumTopicGithubTests(TestCase):
+    def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)